rosie/chamber_of_deputies/classifiers/reimbursement_generalization.py

```diff
@@ -0,0 +1,210 @@
+import os
+import unicodedata
+import numpy as np
+import pandas as pd
+from urllib.request import urlopen
+from sklearn.base import TransformerMixin
+from keras.preprocessing.image import ImageDataGenerator
+from keras.models import Sequential
+from keras.layers import Conv2D, MaxPooling2D
+from keras.layers import Activation, Dropout, Flatten, Dense
+from keras import backend as K
+from keras.callbacks import ModelCheckpoint
+from keras.models import load_model
+from keras.preprocessing.image import img_to_array
+from wand.image import Image
+from PIL import Image as pil_image
+from io import BytesIO
+
+class MealGeneralizationClassifier(TransformerMixin):
+    """
+    Meal Generalization Classifier.
+
+    Dataset
+    -------
+    applicant_id : string column
+        A personal identifier code for every person making expenses.
+
+    category : category column
+        Category of the expense. The model will be applied just in rows where
+        the value is equal to "Meal".
+
+    document_id : string column
+        The identifier of the expense.
+
+    year : string column
+        The year the expense was generated.
+    """
+
+    COLS = ['applicant_id',
+            'document_id',
+            'category',
+            'year']
+
+
+    img_width, img_height = 300, 300
+
+    def train(self,train_data_dir,validation_data_dir,save_dir):
+        #fix random seed for reproducibility
+        seed = 2017
+        np.random.seed(seed)
+
+        nb_train_samples = sum([len(files) for r, d, files in os.walk(train_data_dir)])
+        nb_validation_samples = sum([len(files) for r, d, files in os.walk(validation_data_dir)])
+
+        print('no. of trained samples = ', nb_train_samples, ' no. of validation samples= ',nb_validation_samples)
```

Comment:
Long lines [aren't a good thing](https://www.python.org/dev/peps/pep-0008/#maximum-line-length). This print could be like the following in other to avoid the extra long line:
```
print('no. of trained samples = ', nb_train_samples,
      ' no. of validation samples= ', nb_validation_samples)
```
Extra space was missing there.